vsintegration/tests/unittests/Tests.LanguageService.Colorizer.fs

```diff
@@ -1,1091 +0,0 @@
-// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I think you've moved all these tests across to the Roslyn tests file?  Is there any reason you need to rename the file?  I'd be more comfortable with being able to see a diff on this file which shows which tests have been kept and which haven't.  So perhaps just don't rename the file.